<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wndSendVkFirendAudio</class>
 <widget class="QDialog" name="wndSendVkFirendAudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>474</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Отправить на стену</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="groupUser">
     <property name="title">
      <string>Кому:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QWidget" name="wdgSelectedFriend" native="true">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>3</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>20</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="pixmap">
              <pixmap resource="../../res/res.qrc">:/icons/user-green.png</pixmap>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblFriend">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Имя Пользователя</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="btnChangeFriend">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectFriend">
        <property name="text">
         <string>Выбрать друга...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupAudio">
     <property name="title">
      <string>Аудиозаписи:</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="wdgPlayList" name="lstAudio"/>
      </item>
      <item>
       <widget class="QLabel" name="lblAudioSizeWarning">
        <property name="text">
         <string>&lt;span style='color:red;'&gt;* Вконтакте ограничивает кол-во прикреплённых аудиозаписей до &lt;b&gt;10&lt;/b&gt;, поэтому отмеченные &lt;b&gt;красным&lt;/b&gt; добавлены &lt;b&gt;НЕ будут&lt;/b&gt;.&lt;/span&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupMessage">
     <property name="title">
      <string>Добавить сообщение (необязательно):</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPlainTextEdit" name="txtMessage"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="btnShowMessage">
       <property name="text">
        <string>Добавить сообщение</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>wdgPlayList</class>
   <extends>QTreeView</extends>
   <header location="global">ui/wdgplaylist.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
